app/views/shared/_relation_fields_edit.html.erb

```diff
@@ -49,9 +49,9 @@
     </ol>
     <a href="#" class="add-field">Add related item</a>
   </div>
-  <% desc = "<p>Enter the title of any related item, e.g. a dataset which underpins your publication.</p>
-    <p>Select the option that best describes the relationship between your publication and the item you have indicated as being related.</p>
-    <p>Add an abstract or brief description of the related item.</p>
-    <p>If applicable, enter the DOI or URL of the related item. If the related item does not have a DOI or URL, please give a citation for it.</p>" %>
-  <%= render partial: '/shared/tooltip', :locals => {:tipType => "discoverability" , :tipTitle => "Related publications and dataset" , :tipDescription => desc } %>
+  <% desc = "<p>You can use this field to link your data to any related publications or datasets.</p>
```

Comment:
This file is shared between publications and datasets. You have changed the tooltip so it is relvant to datasets, but this makes it explictly not relevant to publications. You need to make the message dependant on model type here.